setup.py

```diff
@@ -142,7 +142,9 @@ def run(self):
             'six',
             'pytest-xdist',
             'virtualenv',
-        ]
+        ],
+        # Enable pyproject.toml support.
+        'toml': ['tomli; python_full_version<="3.11.0a6"'],
```

Comment:
It seems to me like this would have been written like `'toml': ['coverage[toml]'],` instead (so I don't need to worry about updating this ever).

Reply:
Seems to me you are right :-D

Thanks for the suggestion!